### DISCUSSION TITLE
How to get the original content source in version 8?

### DISCUSSION THREAD

**Discussion by ion-cojocaru** [videojs/video.js#8835] [Q&A]
Hi!
We've recently upgraded our player from version 7 to version 8 and are using videojs-contrib-ads for ad playback. During testing, we encountered an issue where, after an ad finishes, the content doesn't resume correctly because methods like player.src and player.currentSrc are returning the source as a "blob:..." URL instead of the original video source. 
Is there a new method in version 8 for retrieving the original video source that was set?

We have a similar behavior as described here https://github.com/videojs/videojs-contrib-ads/issues/132

If someone encounters a similar issue, a workaround would be to mutate the ads plugin's snapshot by replacing the "blob:..." URL with the original video source.

**Comment by mister-ben** [comment]
I don't think that changed between 7 and 8, either would be a blob URI for MSE playback. But the source object is available in both versions at `player.currentSource()`.
